site/public/css/polls.css

```diff
@@ -43,6 +43,11 @@
     vertical-align: top;
 }
 
+.response {
+    margin-left: 24.2px;
+    text-indent: -24.2px;
+}
```

Comment:
In general, manually specifying widths is a bad idea.  A better option would be to use flexbox to format the columns of inputs and text

Reply:
Buttons and text columns are now organized into table to fix the problem of indenting. Thanks for the suggestion and your template!